acceptance/acceptance_test.go

```diff
@@ -328,7 +339,7 @@ func testAcceptance(t *testing.T, when spec.G, it spec.S) {
 						)
 						txt, err := h.RunE(cmd)
 						h.AssertNotNil(t, err)
-						h.AssertContains(t, txt, "buildpack 'other/stack/bp' version 'other-stack-version' does not support stack 'pack.test.stack'")
+						h.AssertContains(t, txt, "buildpack 'other/stack/bp@other-stack-version' does not support stack 'pack.test.stack'")
```

Comment:
I'm curious about that. I know we use it in the output of `inspect-builder` so my thought is that if we use it in a place where it's intended to be displayed normally surely it would make sense to display it in an error (edge) case.